TAGradingServer/account/admin-hw-report.php

```diff
@@ -6,42 +6,27 @@
 echo <<<HTML
 <div id="container" style="width:100%; margin-top:40px;">
     <div class="modal hide fade in" tabindex="-1" role="dialog" aria-labelledby="Grading Done" aria-hidden="false" style="display: block; margin-top:5%; z-index:100;">
-        <form onsubmit="submitAJAX('ajax/admin-hw-report.php?course={$_GET['course']}&semester={$_GET['semester']}&g_id='); return false;">
+        <form onsubmit="submitAJAX('ajax/admin-hw-report.php?course={$_GET['course']}&semester={$_GET['semester']}'); return false;">
             <div class="modal-header">
                 <h3 id="myModalLabel">Generate Homework Report</h3>
             </div>
 
-            <div class="modal-body" style="padding-top:20px; padding-bottom:20px;">
-                Homework: <select name="gradeable" id='g_id' style="margin-left:5px;">
-                    <option value="-1">No Homework</option>
-HTML;
-                    $params = array();
-                    $db->query("SELECT * FROM gradeable AS g INNER JOIN electronic_gradeable AS eg ON g.g_id=eg.g_id ORDER BY eg_submission_due_date ASC", $params);
-                    $temp = $db->rows();
-                    for($i = 0; $i < count($temp); $i++)
-                    {
-                        $row = $temp[$i];
-                        echo '<option' . ($i == count($temp) -1 ? " selected " : "") . ' value='.$row['g_id'].'>' . $row["g_title"] . '</option>';
-                    }
-echo <<<HTML
-                </select>
-                <div class="loading-gif" style="display:none;"><span style="float: left;margin-left: -100px;margin-top: 10px;">Generating...</span><img src="{$BASE_URL}/toolbox/include/custom/img/loading.gif"></div>
-            </div>
-
             <div class="modal-footer">
-                <div style="float:left; text-align:left;">
+                <!--<div style="float:left; text-align:left;">
                     <label style="display: inline;"><input type="radio" style="margin-top:0" name="type" value='all' checked="checked" /> Grade all submissions for selected and past homeworks</label><br />
                     <label style="display: inline;"><input type="radio" style="margin-top:0" name="type" value='default' /> Fully grade selected homework and regrade past homeworks)</label><br />
                     <label style="display: inline;"><input type="radio" style="margin-top:0" name="type" value='regrade' /> Only regrade selected and past homeworks (ignore new grades)</label><br />
                 </div>
-                <div style="width:50%; margin-left: 50%; margin-top: 20px; float: left;">
+                -->
```

Comment:
Just get rid of this commented out block. We've removed the code it corresponds to so there's no point leaving this commented out in master (as we'll have to dig up this commit anyway if we wanted to revert the changes in the ajax file)